ynput/ayon-deadline
# Refactor Deadline Submission JobInfo

## Changelog Description
This PR tries to make DL submissions easier, mostly handling of `JobInfo` segment. Collection of values is moved out of submission phase much earlier where values could be filled from default values in Settings or overriden by selected list of field by artist in Publisher UI.

Move to universal collector should help with potential validation of filled values. 
Profiles in Settings should allow more control of values (like `priority`, `limit_groups`) based on `task_types` or `hosts` etc.

![jobinfo_settings](https://github.com/user-attachments/assets/363f5a02-4db3-4342-8f64-e7899eb9af8e)
![jobinfo_publisher](https://github.com/user-attachments/assets/23b52c93-7033-4d60-b289-6d3fc2ad7230)


## Additional info
Should be used with https://github.com/ynput/ayon-core/pull/958  !!

All DCC need to be tested:
- [x] Max


## Testing notes:
Should be used with https://github.com/ynput/ayon-core/pull/958 !!
1. do regular DL publishing
2. test that nothing breaks
3. update `ayon+settings://deadline/publish/CollectJobInfo` with new defaults and check if they propagete to Publisher
4. update values in Publisher if they propagate to DL
